### ISSUE TITLE
Request for Oasis-1 Annotated Dataset

### ISSUE THREAD

**Issue by pasha1123** [evanmy/conditional_deformation#1]
I run this code and it works great. I have two queries.
1) In this github, you told us that a numpy file will generate at the end, but in your research paper, you find the Dice score. Can you kindly tell me how to compute the dice score with the help of this code.
2) Can you kindly share the Oasis-1 annotated data set with me as I am a university student and I cannot annotate this dataset without the help of anyone?

**Comment by evanmy** [comment]
Hi, 

Thank you for your interest. The dice loss/score is on the function folder. Essentially, you are computing [dice](https://en.wikipedia.org/wiki/Sørensen–Dice_coefficient) between each region of interest and then averaging the result. Note that in training, we use 1-dice since torch finds the minimum during optimization. 

As for obtaining the labels, there are many different way to do it.  On the paper we used FreeSurfer. You can also run [this trained model](https://github.com/BBillot/SynthSeg), if you are more familiar with neural networks. From my experience, it produces pretty good result.

**Comment by pasha1123** [comment]
Dear Evanmy,
Can you kindly share the files(phenotype.csv, subjectA.npz, etc.) of at least 100 subjects. As I run your code and it gives only 19.2 dice score. It will be a great favor for me.

**Comment by evanmy** [comment]
Apologies for missing the reply. I don't know if I can share the data myself, but you can access them yourself in the webpage. As for reasons you might not achieve the target accuracy, we also preprocessed the data before feeding it into our network. The T1-weighted 3D brain MRI scans was preprocessed using FreeSurfer. This includes skull stripping, bias-field correction, intensity normalization, affine registration to Talairach space, and resampling to 1 mm3 isotropic resolution. The original images had a shape of 256x256x256, which was futher cut using ((48, 48), (31, 33), (3, 29)) along the sides to elimate empty space.

**Comment by bingcheg** [comment]
> futher cut using ((48, 48), (31, 33), (3, 29)) along the sides to elimate empty space.

Hi, 
I want to preprocess my own brain data, can you tell me how to do this step "futher cut using ((48, 48), (31, 33), (3, 29)) along the sides to elimate empty space." ?

**Comment by is-eyu** [comment]
Let `x` be a 3D volume, then you can slice it as `x[48:-48, 31:-33, 3:-29]`
